test/middleware.test.js

```diff
@@ -91,7 +91,7 @@ describe('loopback:middleware generator', function() {
         var newSources = Object.keys(
           readMiddlewaresJsonSync('server')['routes:after']);
         var expectedSources = builtinSources.concat(['my-middleware-4']);
-        expect(newSources).to.have.members(expectedSources);
+        expect(expectedSources).to.have.include.members(newSources);
```

Comment:
`newSources` refers to an array with the name of a single middleware - `my-middleware-4`, which is added for this test.

`expectedSources` refers to an array with the default `loopback#rest` and `my-middleware-4`, which was added for the test.

How do you suggest this assertion be composed?